src/beacon.js

```diff
@@ -67,6 +90,7 @@ export default function Emitter(endpoint, context, blockTransmit) {
         let editedMessage = message;
         editedMessage = message.payload || {};
         editedMessage.type = message.type;
+        editedMessage.client = opts.clientName;
 
```

Comment:
Do we need to send the client type to the `/events` endpoint as well?

Reply:
I wondered about this too. I'll check with @FrazerBayley.

Reply:
I verified with Frazer that the `/events` endpoint does not need the `client` value.